src/core/models/directanswer.js

```diff
@@ -38,4 +38,28 @@ export default class DirectAnswer {
       }
     });
   }
+
+  /**
+   * Applies a formatter to the direct answer if a formatter exists for the direct answer field
+   *
+   * @param {Object.<string, function>} formatters
+   */
+  format (formatters) {
+    if (!this.answer || !this.relatedItem) {
+      return this;
+    }
+
+    const answer = this.answer;
+    const relatedItem = this.relatedItem;
+
+    if (formatters[answer.fieldApiName]) {
+      answer.value = formatters[answer.fieldApiName](
+        answer.value,
+        relatedItem.data.fieldValues,
+        relatedItem.verticalConfigId,
+        true);
+    }
+
+    return this;
```

Comment:
This method mutates the original `DirectAnswer`. Instead of that, we could add `formatters` to the `fromCore` method. Aside from avoiding mutation, it may simplify things if this is handled by `fromCore`. Otherwise, we will need to remember to always call `format` on the result of `fromCore`. What do you think?

Reply:
That makes sense. I'll update